nx.json

```diff
@@ -0,0 +1,46 @@
+{
+  "npmScope": "vitamin-web",
```

Comment:
Does npmScope should'nt be `@vtmn`?

Reply:
Don't really know 😕 I'll switch to `vtmn` anyway you're right


[Docs](https://nx.dev/latest/react/core-concepts/configuration) are not really clear : 


```
NPM Scope

Tells Nx what prefix to use when generating library imports.
```

Reply:
![image](https://user-images.githubusercontent.com/23191482/129487607-4429ede1-5a18-460d-9e5e-78ade959ec0e.png)

I guess it's the scope without the `@`